src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,156 @@
+
+    </Step>
+    <Step>
+
+### Configure Micrometer to forward data to New Relic
+
+Below is an example code snippet that directs Micrometer to use the OpenTelemetry Micrometer bridge. 
+
+This snippet updates your code so OpenTelemetry can detect Micrometer data, then forward that data to New Relic:
+
+```java
+public OpenTelemetry openTelemetry() {
+    return OpenTelemetrySdk.builder()
+        .setMeterProvider(
+            SdkMeterProvider.builder()
+                .setResource(
+                    Resource.getDefault().toBuilder()
+                        .put("service.name", "micrometer-shim")
+                        // Include instrumentation.provider=micrometer to enable micrometer metrics
+                        // experience in New Relic
+                        .put("instrumentation.provider", "micrometer")
+                        .build())
+                .registerMetricReader(
+                    PeriodicMetricReader.builder(
+                            OtlpHttpMetricExporter.builder()
+                                .setEndpoint("https://otlp.nr-data.net")
+                                .addHeader(
+                                    "api-key",
+                                    Optional.ofNullable(System.getenv("NEW_RELIC_LICENSE_KEY"))
+                                        .filter(str -> !str.isEmpty() && !str.isBlank())
+                                        .orElseThrow())
+                                // IMPORTANT: New Relic exports data using delta temporality 
+                                // rather than cumulative temporality
+                                .setAggregationTemporalitySelector(
+                                    AggregationTemporalitySelector.deltaPreferred())
+                                // Use exponential histogram aggregation for histogram instruments
+                                // to
+                                // produce better data and compression
+                                .setDefaultAggregationSelector(
+                                    DefaultAggregationSelector.getDefault()
+                                        .with(
+                                            InstrumentType.HISTOGRAM,
+                                            Aggregation.base2ExponentialBucketHistogram()))
+                                .build())
+                        // Match default micrometer collection interval of 60 seconds
+                        .setInterval(Duration.ofSeconds(60))
+                        .build())
+                .build())
+        .build();
+    }
+```
+    
+        
+    </Step>
+
+        <Step>
+
+### Find your data in New Relic 
+
+Wait a few minutes, trigger some test data, then go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > APM & Services**, then find **Services - OpenTelemetry** to choose the service instrumented with Micrometer.
```

Comment:
Can also mention that they can find metrics using the [metrics explorer](https://docs.newrelic.com/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/).